README_Building.md

```diff
@@ -228,3 +463,49 @@ There's some more information on how to do this on the forum at http://forum.esp
 * Your board will now be flashed
 
 **Note:** if you want to you can change permissions so you don't need `sudo` by typing `sudo cp misc/45-espruino.rules /etc/udev/rules.d;sudo udevadm control --reload-rules` and then re-inserting the board.
+
+----
+
+Building with Bash on Ubuntu on Windows
+---------------------------------------
+
+* Install <https://msdn.microsoft.com/da-dk/commandline/wsl/install_guide>
+* After installation, obtain a bash prompt and use the following, but review the comments:
+
+```
+whoami # installation choice may mean i am root or non-root by default.
+
+sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa # optional, but does no harm
+sudo apt-get update
+sudo apt-get upgrade
+
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash - # install node 
+
+sudo apt-get install -y \
+  build-essential lib32z1 lib32ncurses5 lib32bz2-1.0 lib32ncurses5 ia32-libs git 
+
+# Choose if to use the default ppa repository gcc-arm-embedded. Breaking edge.
```

Comment:
Yes, you could stick this up the top, and just say 'then follow the instructions for Linux'